service/agent/README.md

```diff
@@ -7,8 +7,11 @@ Can be used as alternative to [Chorus proxy](../proxy).
 
 ## Usage
 
-Set proper `url` and `fromStorage` params to [config](./config.yaml) and deploy agent to be reachable by source s3
-storage.
+Set proper `url`, `fromStorage`, and `redis` credentials to [config](./config.yaml) and deploy agent to be reachable by source s3
+storage. 
+
+> [!IMPORTANT]  
+> Make sure that `chorus-worker` and `chorus-agent` using the same Redis instance.
```

Comment:
Better:
Make sure that `chorus-worker` and `chorus-agent` are using the same Redis instance.

Reply:
done